test/metabase/channel/template/handlebars_test.clj

```diff
@@ -0,0 +1,54 @@
+(ns metabase.channel.template.handlebars-test
+  (:require
+   [clojure.java.io :as io]
+   [clojure.test :refer :all]
+   [metabase.channel.template.handlebars :as handlebars]
+   [metabase.util :as u])
+  (:import
+   (com.github.jknack.handlebars
+    Helper)))
+
+(set! *warn-on-reflection* true)
+
+(def custom-hbs (doto (handlebars/registry (handlebars/classpath-loader "/" ""))
```

Comment:
nit: linebreak before doto plz

Reply:
hehe sure, lemme do that in the next PR :)